ruby-challenges.rb

```diff
@@ -22,7 +34,14 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def no_vowels string
+    string.delete "aeiou"
 
+end
```

Comment:
Something tells me you got "Abby Rd"... capital and lowercase are different characters